src/web/components/Nav/ExpandedMenu/ExpandedMenu.tsx

```diff
@@ -95,7 +97,7 @@ export const ExpandedMenu: React.FC<{
 	nav: NavType;
 }> = ({ display, nav }) => {
 	return (
-		<div id="expanded-menu">
+		<div id="expanded-menu-root">
```

Comment:
`expanded-menu` ID was already being used

Reply:
Ah, this was being flagged up by accessibility checkers - I've been wanting to fix this for ages!